xcp_d/utils/bids.py

```diff
@@ -152,12 +152,21 @@ def collect_data(
     layout : pybids.layout.BIDSLayout
     subj_data : dict
     """
-    layout = BIDSLayout(
-        str(bids_dir),
-        validate=bids_validate,
-        derivatives=True,
-        config=["bids", "derivatives"],
-    )
+    if input_type != 'hcp':
+        layout = BIDSLayout(
+            str(bids_dir),
+            validate=bids_validate,
+            derivatives=True,
+            config=["bids", "derivatives"],
+        )
```

Comment:
I believe that just about everything that applies to "hcp" data will apply to "dcan" data as well.